### DISCUSSION TITLE
JSON import/export vs js migrations?

### DISCUSSION THREAD

**Discussion by benallfree** [pocketbase/pocketbase#1891] [Q&A] [answered]
Apologies if this has been asked, I couldn't find it in the discussions.

1. How to create a JS migration?
2. I have been importing a JSON schema, but that posing some issues that I think migrations would solve. Things like mismatched collection IDs. Are there any plans to add an 'export JS migration' tool to the admin?

**Comment by MRSessions** [comment, marked as answer]
Wanted to throw this out there. I ended up using the framework setup using Go, and created my migrations that way. While I was searching, I did stumble across a couple of repos that had some JS migrations in them. One of them is hosted on your awesome-pocketbase repo.

https://github.com/spinspire/pocketbase-sveltekit-starter/tree/master/backend

You can see their migrations in the /pb_migrations folder.

The Maintainer has stated that he is working on documentation for JS migrations here: https://github.com/pocketbase/pocketbase/discussions/1464

It does show some basic examples.

Hopefully we'll see the docs soon :)
